core/string/ustring.cpp

```diff
@@ -2575,6 +2575,72 @@ int64_t String::to_int(const wchar_t *p_str, int p_len) {
 	return integer * sign;
 }
 
+uint64_t String::to_uint64(bool p_clamp) const {
+	int str_length = length();
+	if (str_length == 0) {
+		return 0;
+	}
+
+	int start = 0;
+	// Skip leading whitespace
+	while (start < str_length &&
+			(is_whitespace(operator[](start)) || operator[](start) == '+')) {
```

Comment:
This will happily convert strings like `+++5` to 5. Is this wanted behavior?

Reply:
thought it kept the code cleaner and it seemed like a benign edge case.

Reply:
I think it's good to be correct even in "benign" edge cases because sooner or later, somebody will run into these and then the function has to be changed.

Reply:
The other code explicitly supports -----5, so it would be weird to not support +++++5.